features/account/customer_account/subscribing_newsletter.feature

```diff
@@ -6,10 +6,10 @@ Feature: Subscribing to the newsletter
 
     Background:
         Given the store operates on a single channel in "United States"
-        And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "whitehouse"
```

Comment:
Is this change needed here?

Reply:
Yes and not. Our behat api security service has hardcoded `sylius` password. This way was faster but I can change it 🚀

Reply:
Is the password even needed in this scenario?

Reply:
👏